spec.html

```diff
@@ -35844,7 +35844,7 @@ <h1>
         <!-- CharacterClassEscape -->
         <emu-grammar>CharacterClassEscape :: `d`</emu-grammar>
         <emu-alg>
-          1. Return the ten-element CharSet containing the characters `0` through `9` inclusive.
+          1. Return the ten-element CharSet containing the characters whose mathematical values are 0x30, 0x31, 0x32, 0x33, 0x34, 0x35, 0x36, 0x37, 0x38, and 0x39.
```

Comment:
Listing the code point values is also less clear than what was here originally. If you want to explicitly list the decimal digits, say "the characters `0`, `1`, `2`, `3`, `4`, `5`, `6`, `7`, `8`, `9`", not this.